### ISSUE TITLE
UX: colors of instructions

### ISSUE THREAD

**Issue by Sohalt** [git-learning-game/oh-my-git#34]
The instructions are printed as white on red, which kind of reads like a warning. A more neutral background would be better suited imho.
Also you could add bullet points of some sort, to make it more clear that that is a list of steps. Right now it has the same "rectangle with rounded corners" look as buttons or the file browser.

**Comment by blinry** [comment]
Yeah, that's really unfortunate. It might be ideal to make it look like a "checklist"!
